test/reductions/vass/reductions-perf.good

```diff
@@ -0,0 +1,6 @@
+4 locale(s)   dist: Block  sizes: 4000000 16384 16384 15625  reps: 10
```

Comment:
Oh duh. Sorry for some reason I read it as reductions.perf.good and thought the "perf" part was an extension instead of the basename. Ignore me